app/ApolloClientProvider.tsx

```diff
@@ -1,24 +1,20 @@
 'use client';
 
-import {
-  ApolloClient,
-  ApolloLink,
-  HttpLink,
-  SuspenseCache,
-} from '@apollo/client';
+import { ApolloLink, HttpLink, SuspenseCache } from '@apollo/client';
 import {
   ApolloNextAppProvider,
+  NextSSRApolloClient,
```

Comment:
To solve the error with deployment because of `Next SSR` on the screenshot below
I am using the `NextSSRApolloClient` instead of the `ApolloClient`

`Error: When using Next SSR, you must use the 'NextSSRApolloClient*`
![Screenshot 2023-07-03 at 11 52 11](https://github.com/upleveled/graphql-example-spring-2023-austria-vienna/assets/74430629/0aeb1c8c-9009-48b2-aa22-623f07a35685)